build/config.props

```diff
@@ -66,8 +66,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <NUGET_PFX_PATH>$(MSBuildThisFileDirectory)\..\keys\NuGetKey.snk</NUGET_PFX_PATH>
-    <MS_PFX_PATH>$(MSBuildThisFileDirectory)\..\keys\35MSSharedLib1024.snk</MS_PFX_PATH>
```

Comment:
[`MsBuildThisFileDirectory`](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-reserved-and-well-known-properties?view=vs-2019) already includes the trailing backslash.

Reply:
Were the paths being resolved correctly prior to this change?

Reply:
Yes, just an extra slash.

    /keyfile:E:\A\_work\68\s\build\\..\keys\35MSSharedLib1024.snk